waves/src/App.tsx

```diff
@@ -229,6 +223,18 @@ function App() {
         state.wallet.status.loaded ? 5000 : null,
     );
 
+    state.rate = useSSE("rate", {
```

Comment:
Uhm, why are you overwriting the `state` variable here? With reducers, the `state` should be read-only!

Reply:
see discussion below.